clients/java/src/main/java/io/odpf/raccoon/model/Response.java

```diff
@@ -0,0 +1,38 @@
+package io.odpf.raccoon.model;
+
+import java.util.Map;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+
+/*
+ * Response holds the information about the request.
+ */
+public class Response {
+
+  /**
+   * status denotes status of the request.
+   */
+  private int status;
+
+  /**
+   * code gives more detail of what happened to the request.
```

Comment:
Should we explain what are the different codes (from the proto)?
Similar comment for the status as well